botframework.py

```diff
@@ -255,16 +226,22 @@ def get_botframework():
         @flask_app.route('/botframework', methods=['POST'])
         def post_botframework():
             req = request.json
+
+            req['text'] = unicodedata.normalize("NFKD", req['text']).encode('ascii', 'ignore').decode('UTF-8')
```

Comment:
not clear why this is needed

Reply:
You have a very valid point. We added a new function with a self-describing name and some documentation. Thanks for the feedback